updating/index.adoc

```diff
@@ -0,0 +1,75 @@
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-unpause_update-using-custom-machine-config-pools[Unpausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp-remove_update-using-custom-machine-config-pools[Moving a node to the original machine config pool]
+
+['updating-clusters-overview-update-cluster-with-rhel-compute-machines']
+== Updating a cluster that includes RHEL compute machines
+xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]:  You can update an {product-title} cluster. If your cluster contains {op-system-base-full} machines, you must update those machines. You can perform the following actions:
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on RHEL machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating RHEL compute machines in your cluster]
+
+['updating-clusters-overview-update-restricted-network-cluster']
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. For more information, check the following items:
+
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-mirror-host[Preparing your mirror host]
+* xref:../updating/updating-restricted-network-cluster.adoc#installation-adding-registry-pull-secret_updating-restricted-network-cluster[Configuring credentials that allow images to be mirrored]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-restricted_updating-restricted-network-cluster[Updating the restricted network cluster]
+* xref:../updating/updating-restricted-network-cluster.adoc#images-configuration-registry-mirror_updating-restricted-network-cluster[Configuring image registry repository mirroring]
+* xref:../updating/updating-restricted-network-cluster.adoc#generating-icsp-object-scoped-to-a-registry_updating-restricted-network-cluster[Widening the scope of the mirror image catalog to reduce the frequency of cluster node reboots]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-install[Installing the OpenShift Update Service Operator]
```

Comment:
There are two parts in [this page](https://docs.openshift.com/container-platform/4.8/updating/updating-restricted-network-cluster.html):
1. Updating a restricted network cluster using the CLI
2. Updating a restricted network cluster using the OpenShift Update Service

We seems missing the summary/brief description about part 2 at the [beginning](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/index.adoc#updating-a-restricted-network-cluster), and only links here will mislead users. I suggest we should also add a brif description before the detail steps as following:
```
[Installing the OpenShift Update Service Operator](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-install)

[Creating an OpenShift Update Service application](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-create-service)

[Deleting an OpenShift Update Service application](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-delete-service)

[Uninstalling the OpenShift Update Service Operator](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-uninstall)
```

Reply:
Addressed- Added the sentence which implies that customers can update using the Openshift update service.

Reply:
ack